src/dcc/Defs.hxx

```diff
@@ -50,6 +50,11 @@ enum class TrainAddressType : uint8_t
     DCC_LONG_ADDRESS,
     /// Marklin-motorola packets. Addresses 1..80 are supported.
     MM,
+    /// Per-output addressing of DCC basic accessories. (user facing is 1 to
+    /// 2047, internal is 0 to 2047)
+    DCC_ACCY_BASIC_OUTPUT,
+    /// DCC 11-bit extended accessory decoder address (1 to 2047).
```

Comment:
(1 to 2048).

Reply:
I replaced this with the text from basic accy: 
>  (user facing is 1 to 2048, internal is 0 to 2047)